### ISSUE TITLE
Change django-cron over to just a management command

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#664]
Thank you for contributing to this project!

## Describe your problem or feature you'd like added
At Michigan, we're not using the Django cron features. [This project](https://github.com/Tivix/django-cron) hasn't been updated in over a year and has a number of open issues including one I opened where it doesn't capture the entire log file. I feel like it's easier if we just remove it and suggest better ways of doing this cron.

## Describe the solution you'd like
Just use a management command to run the cron job(s)

Are you using this @andrew-gardener or anyone else? Would this be a problem for you? The only thing this really is useful for at the moment is the logs in the database and whether or not the job ran correctly, but I think we can capture this status just with the regular unix cron command. We're actually running this job now in Jenkins.

**Comment by andrew-gardener** [comment]
I'm fine with removing django-cron. We were going to try and schedule the cron in other infrastructure anyways (and for development, we always force run as needed).

**Comment by jonespm** [comment]
Yeah, I think everyone's fine with removing it, it just adds extra complexity and I personally find launching this job from Jenkins (or any other job scheduler) to be to much better and afford many more options.

**Comment by jonespm** [comment]
If we removed this we **probably** want to keep the [CronJobLog](https://github.com/Tivix/django-cron/blob/master/django_cron/models.py) and probably have it as a different name. We might just be able to write a migration to `RENAME TABLE` after we add our local model.

**Comment by jonespm** [comment]
This process also creates a "lock" entry in the cache table like.

`myla:1:DashboardCronJob
`
This is actually nice, but also annoying since the only way to clear the lock is to wait till it expires or manually go in and delete the row.

**Comment by jennlove-um** [comment]
@jonespm Is this something we should still consider doing?

**Comment by jonespm** [comment]
I think this could be nice to do still. But also if we just upgrade the django-cron #1400 and have a way to clear the lock it would be fine too. They've done some nice updates in the recently with that library so moving completely away probably isn't necessary and we'd still want a few things from that library. 

I'd leave this open for now until we see what happens with those related issues.

**Comment by jennlove-um** [comment]
@jonespm Is this something we should still consider doing since we upgraded django-corn?
